### PR TITLE
chore(Asset Submission): Update submission block strings

### PR DESCRIPTION
**Pull request by getflourish**
- Updates according to feedback from Peige: https://app.clickup.com/t/861mz0g55
- pluralisation of “file”

### CODE REVIEW

File: packages/guest-submissions-block/src/Components/Headline/ModalHeadline.tsx [hunk 1]
```diff
@@ -22,7 +22,7 @@ export const ModalHeadline: FC<BlockProps> = ({ appBridge }) => {
             content={modalcontent}
             onTextChange={onTextChange}
             defaultValue={''}
-            placeholder={'note / description'}
+            placeholder={'Instructional text'}
```
Comment (marguslant0) [inline packages/guest-submissions-block/src/Components/Headline/ModalHeadline.tsx:25]:
```suggestion
            placeholder="Instructional text"
```

File: packages/guest-submissions-block/src/View/SuccessPage.tsx [hunk 2]
```diff
@@ -22,15 +23,15 @@ export const SuccessPage: FC<BlockProps> = ({ appBridge }) => {
...
+                        {uploadedFiles || 0} {pluralizeFile(uploadedFiles || 0)} submitted successfully!
                     </Text>
                     <RichText
                         isEditing={isEditing}
                         appBridge={appBridge}
                         content={successContent}
                         onTextChange={onTextChange}
                         defaultValue={DEFAULT_SUCCESS_MESSAGE}
-                        placeholder={'Add Here your personlized Succes Message'}
+                        placeholder={'Description'}
```
Comment (marguslant0) [inline packages/guest-submissions-block/src/View/SuccessPage.tsx:34]:
```suggestion
                        placeholder="Description"
```

File: packages/guest-submissions-block/src/Components/Headline/ModalHeadline.tsx (hunk 1 above)
Comment (getflourish) [reply packages/guest-submissions-block/src/Components/Headline/ModalHeadline.tsx:25]: Correct: we can treat this prop as a plain string. But, looking towards i18n, we’d probably end up needing the curly braces again :D

Comment (getflourish) [reply packages/guest-submissions-block/src/Components/Headline/ModalHeadline.tsx]: I’ll change it to be correct *now*.

**Comment by getflourish** [comment]
Will delete this branch as suggested by @julianiff and commit to the base branch instead

**Pull request closed without merging**